src/modules/gameDevs/gameDevWebhooksRoutes.ts

```diff
@@ -77,7 +83,7 @@ export const postLeaderboardResultRoute: ServerRoute = {
         assign: 'webhookValidation',
       },
       {
-        method: parseWebhookPayload,
+        method: parseWebhookPayload(postLeaderboardResultScoreResquestSchema),
```

Comment:
made an enhancement on parseWebhookPayload so we can reuse  to validate schema on other routes

Reply:
Beautiful